src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1208,7 +1218,25 @@ void ImportStoreIndirect(TypeDesc type)
 
         void ImportThrow()
         {
-            throw new NotImplementedException();
+            var value = Pop();
+
+            var obj = _typeSystemContext.GetWellKnownType(WellKnownType.Object);
+            var exception = _typeSystemContext.GetWellKnownType(WellKnownType.Exception);
```

Comment:
> we also plan to add a parameter to check for CLS compliance

I do not think that CLS compliance is that interesting for IL verification. Maybe the few places like this one - where the IL is valid, but suspect - can be optional warnings.

However, I think it would be very useful to distinguish between IL validation (e.g. verifies valid IL, but does not verify the strict type safety rules) and IL verification (verifies strict type safety rules as well).